cloudrun/Dockerfile

```diff
@@ -22,18 +22,16 @@ ENV CLOUDSDK_CONFIG="/gcp/config/application_default_credentials.json"
 
 CMD exec flask run -p 5000 -h 0.0.0.0
 
-
-FROM base as remote_dev
+FROM base AS remote_dev
 
 RUN apt install -y git
 
-
-FROM base as prod
+FROM base AS prod
 
 COPY . .
 
 RUN pip install .
 RUN pip install arcgis==2.*
 RUN pip install gunicorn
 
-CMD exec gunicorn --bind :$PORT --timeout 0 download.main:app
+CMD ["exec", "gunicorn", "--bind", ":$PORT", "--timeout", "0", "download.main:app"]
```

Comment:
**question**: the exec piece may be unnecessary. I'm not as confident about this update.

Reply:
It sounds like [both forms are supported](https://docs.docker.com/reference/dockerfile/#shell-and-exec-form). The shell form seems easier to read and maintain. But I don't have any major problem with it.

Reply:
Yeah i'm not sure what issues it can solve but the warning and fix seemed easy enough that I thought we should include it.